cellbase-lib/src/main/java/org/opencb/cellbase/lib/builders/clinical/variant/GwasIndexer.java

```diff
@@ -56,36 +59,31 @@ public GwasIndexer(Path gwasFile, Path dbSnpTabixFile, Path genomeSequenceFilePa
     }
 
     public void index() throws RocksDBException, IOException {
-        logger.info("Parsing GWAS catalog file ...");
-
-        BufferedReader inputReader = null;
-        TabixReader dbsnpTabixReader = null;
-
-        try {
-            logger.info("Opening GWAS catalog file " + gwasFile + " ...");
-            inputReader = new BufferedReader(new FileReader(gwasFile.toFile()));
+        try (BufferedReader inputReader = new BufferedReader(new FileReader(gwasFile.toFile()));
+             TabixReader dbsnpTabixReader = new TabixReader(dbSnpTabixFile.toString())) {
 
             logger.info("Ignoring GWAS catalog file header line ...");
-            String line = inputReader.readLine();
+            inputReader.readLine();
+            ++lineCounter;
 
+            Map<String, String> chromosomeMap = buildChromosomeMap(dbsnpTabixReader);
             Map<String, GwasAssociation> gwasMap = new HashMap<>();
-            logger.info("Opening dbSNP tabix file " + dbSnpTabixFile + " ...");
-            dbsnpTabixReader = new TabixReader(dbSnpTabixFile.toString());
 
             long processedGwasLines = 0;
 
-            logger.info("Parsing GWAS catalog file ...");
+            logger.info("Parsing GWAS catalog file {} ...", gwasFile);
+            String line;
             while ((line = inputReader.readLine()) != null) {
+                ++lineCounter;
                 if (!line.isEmpty()) {
                     processedGwasLines++;
```

Comment:
Might want to use a `org.opencb.commons.ProgressLogger` here

```java
ProgressLogger progressLogger = new ProgressLogger("Lines parsed").setBatchSize(10000);

...
if (!line.isEmpty() ) {
  progressLogger.increment(1);
}
```

Reply:
This is just a comment, no action is required (i.e. it's optional)

Reply:
Hmm, that's useful, but I can't find a method to get the current number of parsed lines. I need this information to include in the log messages when parsing fails.

Reply:
That is very inconvenient... there is no way of obtaining that..